### PR TITLE
feat: initial blank implementation of svc-compliance

### PR DESCRIPTION
**Pull request by romanmandryk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,30 +1,30 @@
-## [Release 0.2.0-develop.2](https://github.com/Arrow-air/svc-template-rust/releases/tag/v0.2.0-develop.2)
+## [Release 0.2.0-develop.2](https://github.com/Arrow-air/svc-compliance/releases)
```
Comment (amsmith-pro) [inline CHANGELOG.md]: @owlot should he empty this Changelog?

**Review by amsmith-pro — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (owlot) [reply CHANGELOG.md]: Yes! it should start off empty indeed

**Comment by amsmith-pro** [comment]
Nice. Will be cool to see this component in the flow

File: server/Cargo.toml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+description = "Arrow compliance service GRPC server"
 edition     = "2021"                                                       # Allowed values are 2015, 2018, 2021
 homepage    = "https://www.arrowair.com/docs/documentation/services/intro"
 keywords    = ["vtol", "server", "template"]                               # max 5
 license     = "BUSL-1.1"
-name        = "svc-template-rust"
-repository  = "https://github.com/Arrow-air/svc-template-rust"
+name        = "svc-compliance"
+repository  = "https://github.com/Arrow-air/svc-compliance"
 version     = "0.2.0-develop.2"
```
Comment (owlot) [inline server/Cargo.toml]: this should start at 0.0.1

File: client-grpc/Cargo.toml [hunk 3]
```diff
@@ -4,8 +4,8 @@ edition     = "2021"                                                       # All
 homepage    = "https://www.arrowair.com/docs/documentation/services/intro"
 keywords    = ["vtol", "client", "template"]                               # max 5
 license     = "BUSL-1.1"
-name        = "svc-template-rust-client-grpc"
-repository  = "https://github.com/Arrow-air/svc-template-rust"
+name        = "svc-compliance-client-grpc"
+repository  = "https://github.com/Arrow-air/svc-compliance"
 version     = "0.2.0-develop.2"
```
Comment (owlot) [inline client-grpc/Cargo.toml]: version should start at 0.0.1

**Comment by amsmith-pro** [comment]
Added L4-SCHEDULER.53 to the requirements, svc-scheduler's SDD diagrams will need to be updated for the new communications with svc-compliance

**Review by amsmith-pro — APPROVED** [review]

**Pull request merged**